packages/block-editor/src/utils/get-editor-region.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * Gets the editor region for a given editor canvas element or
+ * returns the passed element if no region is found
+ *
+ * @param { Object } editor The editor canvas element.
+ * @return { Object } The editor region or given editor element
+ */
+export default function getEditorRegion( editor ) {
+	if ( ! editor ) {
+		return null;
```

Comment:
Note that this could be returning `null`, but you're calling focus() unconditionally.

Reply:
Good catch! Fixed here: https://github.com/WordPress/gutenberg/pull/62980